lib/rubocop/cop/rails/render_plain.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop identifies places where `render text:` can be
+      # replaced with `render plain:`.
+      #
+      # @example
+      #   # bad - sets MIME type to `text/html`
+      #   render text: 'Ruby!'
+      #
+      #   # bad - requires explicit MIME type declaration
+      #   render text: 'Ruby!', content_type: 'text/plain'
+      #
+      #   # good
```

Comment:
I think that bringing notes from the guide will help users who use this rule :-)

```suggestion
      #   # good - short and precise
```

cf: https://rails.rubystyle.guide/#plain-text-rendering